plugins/linear-genome-view/src/LinearGenomeView/index.test.ts

```diff
@@ -304,8 +302,8 @@ test('can navToMultiple', () => {
     { refName: 'ctgA', start: 5000, end: 10000 },
     { refName: 'ctgC', start: 0, end: 5000 },
   ])
-  expect(model.offsetPx).toBe(2793)
-  expect(model.bpPerPx).toBeCloseTo(12.531)
+  expect(model.offsetPx).toBe(199)
```

Comment:
snapshot updated in response to testing in the browser. the behavior is ultimately ambiguous as the regions requested exist twice in the displayed regions. the new code "prefers the start of the displayed regions" while the old "prefers the end of the displayed regions"

this branch:
![image](https://user-images.githubusercontent.com/6511937/232613810-0f7c34f1-6c8e-49fb-8b7b-54351f6c0f0f.png)


main:

![image](https://user-images.githubusercontent.com/6511937/232613195-cbc4e667-6da2-47f7-9129-4026f6d369f3.png)